packages/zoe/test/unitTests/contracts/escrowToVote.js

```diff
@@ -0,0 +1,118 @@
+/* global harden */
+// @ts-check
+
+// Eventually will be importable from '@agoric/zoe-contract-support'
+import { assert } from '@agoric/assert';
+import makeStore from '@agoric/store';
+import { makeZoeHelpers } from '../../../src/contractSupport';
+
+/**
+ * Implement coin voting. Give a voting capability when a payment is
+ * escrowed.
+ *
+ * @typedef {import('../../../src/zoe').ContractFacet} ContractFacet
+ * @typedef {import('@agoric/ERTP').Amount} Amount
+ * @param {ContractFacet} zcf
+ */
+const makeContract = zcf => {
+  const { assertKeywords, checkHook } = makeZoeHelpers(zcf);
+  assertKeywords(harden(['Assets']));
+  const {
+    terms: { question },
+  } = zcf.getInstanceRecord();
+
+  const offerHandleToResponse = makeStore('offerHandle');
+
+  // `question` is a string defined in the terms.
+  // We assume the only valid answers are 'YES' and 'NO'
+  const validateResponse = response => {
+    assert(
+      Object.keys(response).length === 1 &&
```

Comment:
I do not like using an arbitrary user-provided string as a computed property name. What if the question is `'toString'`? We decided to tolerate user-provided zoe keywords only because we imposed a rule that guaranteed non-collision. I doubt we want to do likewise for these questions.

Reply:
Ok, that's a good point. Let me see what I can do

Reply:
I will leave out the question in the response, such that the voter only submits 'YES' or 'NO'